tests/unit/plugins/modules/source_control/github/test_github_repo.py

```diff
@@ -17,42 +17,42 @@ def debug_mock(url, request):
     print(request.original.__dict__)
 
 
-@urlmatch(netloc=r'api\.github\.com:443$', path=r'/orgs/.*', method="get")
+@urlmatch(netloc=r'api\.github\.com.*$', path=r'/orgs/.*', method="get")
```

Comment:
that's probably not a good idea, it would match the hostname `api.github.comrade-dr-evil.blablabla.com`

Reply:
well, it's test code so it won't matter all that much... :)

Reply:
Now it should be OK :)

Reply:
LGTM